### PR TITLE
Add missing spaces around link in FAQ section.

### PR DESCRIPTION
**Pull request by jimender2**
Simple fix here to add some missing spaces.  I think I followed the contributing guide correctly but if I did not, please let me know.

image

Resolves #

## Target Release

1.14.7

### CODE REVIEW

**Review by JanMa — APPROVED** [review]
Lgtm, thanks @jimender2

File: website/src/components/FAQSection/index.tsx [hunk 1]
```diff
@@ -42,8 +42,8 @@ const accordionData: AccordionItem[] = [
         prior to the BUSL. This corresponds to a few commits after 1.14.8,
         but prior to 1.14.9 being cut.
 
-        From here forward, we'll be adhering to our
-        <a href="/docs/policies/migration/#proposal">compatibility policy</a>
+        From here forward, we'll be adhering to our{" "}
+        <a href="/docs/policies/migration/#proposal">{" "}compatibility policy</a>
```
Comment (cipherboy) [inline website/src/components/FAQSection/index.tsx:46]:
```suggestion
        From here forward, we'll be adhering to our{" "}
        <a href="/docs/policies/migration/#proposal">compatibility policy</a>{" "}
```

Doesn't the current version mean we'll end up with two spaces in front and none behind?

Comment (JanMa) [reply website/src/components/FAQSection/index.tsx:46]: It does, you're right.

**Comment by jimender2** [comment]
Thanks @cipherboy for that catch.

**Pull request merged**